dev/MRTCore/azure-pipelines.yml

```diff
@@ -250,13 +250,31 @@ jobs:
 
 # START of copying files into their final package location.
 
+  # Place Microsoft.ApplicationModel.Resources.winmd in the Microsoft.ApplicationModel.Resources\
```

Comment:
@huichen123 the corresponding ADO change: https://microsoft.visualstudio.com/ProjectReunion/_git/ProjectReunion/pullrequest/5623504